### PR TITLE
[LIEFEREMMA] Added grpc-web-proxy

### PR DESCRIPTION
**Pull request by strobsi**
- Added grpc-web proxy submodule
- Added grpc-web container to docker-compose

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -8,6 +8,14 @@ services:
     networks:
       - postgres
 
+  grpc-web-proxy:
+    build: ./grpc-web-proxy/
+    ports:
+      - "8080:8080"
+      - "9901:9901"
+    networks:
+      - postgres
```
Comment (marcelbuesing) [inline docker-compose.yml]: Probably should not be part of the postgres network. I assume we should have a separate "frontend" or "api" network. I think the biggest benefit of the networks is basically only a limited number of services have access to the database (assuming that unlike now in production we do not expose the port).

Comment (strobsi) [reply docker-compose.yml]: Fixed now. I use "api" network as it represents the purpose best. I also removed the admin port `9901`

**Comment by marcelbuesing** [comment]
Backend should probably be part of api network as well

**Comment by strobsi** [comment]
> Backend should probably be part of api network as well

Added 'api' network to lieferemma backend container in compose file.

**Pull request merged**